src/web/lib/components/BrowserPreview/index.js

```diff
@@ -10,16 +10,37 @@ const bgImages = require.context('../../../../images/', false, /bg-.*\.png/);
 const buttonImages = require.context('../../../../images/', false, /.*-16\.svg/);
 
 export const BrowserPreview = ({
+  size,
   theme,
   setSelectedColor,
-  selectedTab = 0
+  selectedTab = 0,
+  children = null,
+  setTheme = null
 }) => {
   const clickSelectColor = name => e => {
```

Comment:
@lmorchard i'm not sure if there's a better way to selectively bind click handlers. I guess i could do the logic in the JSX, but i'm not sure quite how that would look.